Services/UIComponent/Toolbar/classes/class.ilToolbarGUI.php

```diff
@@ -333,6 +328,7 @@ public function getHTML(): string
                     $markup_items .= $tpl_separator->get();
                 }
                 foreach ($group as $item) {
+                    $tpl_items->touchBlock("item");
```

Comment:
To be honest, I do not understand why it needs this touchBlock at this front position to work. If I put the touchBlock last, some empty items are rendered; if I don't put it here, many items are missing. Can someone please explain this to me (@nhaagen)?